--- foomatic-db-20130115/Makefile.in.orig	2013-01-15 21:17:08.472826846 +0100
+++ foomatic-db-20130115/Makefile.in	2013-01-15 21:18:48.057630733 +0100
@@ -68,7 +68,10 @@ install-db:
 	install -d $(DESTDIR)$(LIBDIR)/db/source/PPD
 	install -d $(DESTDIR)$(LIBDIR)/xmlschema
 	cp db/oldprinterids $(DESTDIR)$(LIBDIR)/db
-	cp db/source/printer/*.xml $(DESTDIR)$(LIBDIR)/db/source/printer
+	( cd db/source/printer/; \
+	  for xml in `find -name "*.xml`; do \
+	    cp $$xml $(DESTDIR)$(LIBDIR)/db/source/printer; \
+	  done )
 	( cd db/source/driver/; \
 	  for d in $(DRIVERXMLS); do \
 	    cp $$d $(DESTDIR)$(LIBDIR)/db/source/driver; \
